src/flamegraph/mod.rs

```diff
@@ -394,6 +394,8 @@ where
     W: Write,
 {
     let mut reversed = StrStack::new();
+    let lines = lines.into_iter().filter(|line| !(line.is_empty() || line.starts_with("# ")));
```

Comment:
I think this is missing a `.trim()` call. Without that, a line that's empty except for spaces would not get skipped whereas it does today.

Reply:
Ah yes, fixed :)